CareKitStore/CareKitStore/Structs/OCKScheduleElement.swift

```diff
@@ -88,7 +88,9 @@ public struct OCKScheduleElement: Codable, Equatable {
         }
     }
 
-    private let calendar: Calendar
+    private static var calendar: Calendar = {
+        Calendar.current
+    }()
```

Comment:
Let's avoid storing the default calendar here in case it changes after this schedule element is created. (Though we likely need some other improvements here to handle time zone changes)

```suggestion
    private var calendar: Calendar { .current }
```